src/extension/options-page/DashboardDialogs/Contact.tsx

```diff
@@ -60,22 +65,34 @@ export function DashboardContactDeleteConfirmDialog(
     )
 }
 
-export function DashboardContactDialog(props: WrappedDialogProps<ContactProps & { onUpdated: () => void }>) {
+export function DashboardContactDialog(
+    props: WrappedDialogProps<
+        ContactProps & {
+            onUpdated: (mutatedContact: MutatedContact) => void
+        }
+    >,
+) {
     const { t } = useI18N()
     const classes = useStyles()
     const { contact, onUpdated } = props.ComponentProps!
     const [nickname, setNickname] = useState(contact.nickname)
     const [avatarURL, setAvatarURL] = useState(contact.avatar)
-
-    // TODO: revalidate SWR after this change
```

Comment:
remove this TODO comment, since this PR had done the revalidate: 

```ts
mutate(mutatedData, false)
revalidate()
```

You can verify it indeed revalidates by giving mutation a wrong nickname, you will see it would be corrected.
```ts
Services.Identity.updateProfileInfo(contact.identifier, { 
  nickname: nickname + 'abc', avatarURL, forceUpdateAvatar: true
}),
```